libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
```

Comment:
Please do. The fact that the list of tags which are tied to text will be in the function is another good reason to put it in Text